__tests__/action_single_baseline_preview.test.js

```diff
@@ -0,0 +1,251 @@
+const action = require("../src/action");
+const core = require("@actions/core");
+const github = require("@actions/github");
+
+jest.mock("@actions/core");
+jest.mock("@actions/github");
+
+describe("Single report", function () {
+  let createComment;
+  let listComments;
+  let updateComment;
+  let output;
+
+  function getInput(key) {
+    switch (key) {
```

Comment:
This would be more readable as a simple const hash.

Reply:
Done